CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+# 3.0.0 (2022-08-TBD)
+All the changes are introduced by SDK V3, please check [Migrating to the AWS SDK for JavaScript V3](https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/migrating-to-v3.html) to learn how to migrate to the AWS SDK for JavaScript V3 from AWS SDK for JavaScript V2.  
+
+## :tada: Enhancements
+* Migrated to [AWS SDK for JavasScript V3](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/index.html).
+
+## :boom: Breaking changes
+* Updated driver to comply with new [service exception class](https://aws.amazon.com/blogs/developer/service-error-handling-modular-aws-sdk-js/) and new [modular session APIs](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-qldb-session/classes/qldbsession.html).
```

Comment:
Should we provide specific guidance on how to transition customers to use v3? Maybe we can include a note to separate `qldbClientOptions` and `httpOptions` in their QldbDriver constructors.

Reply:
Nice call out. What about adding this:

-  Changed driver constructor to take a new type of `qldbClientOptions` and added a new parameter to configure HTTP Client. Application code need to be modified to separate `qldbClientOptions` and `httpOptions` in `QldbDriver` constructors.

Reply:
Found a bug here https://github.com/awslabs/amazon-qldb-driver-nodejs/pull/95/files#diff-1799ddf63dff57f460ced3b0e767ea5493d94150ddbdbeb8bbf97ceccb446cb3L87. We didn't configure the low-level SDK with customized `httpOptions`. Will take a look.